lighty-models/test/lighty-test-models/src/main/yang/test-models@2018-01-19.yang

```diff
@@ -33,6 +33,28 @@ module test-models {
         }
     }
 
+    container choice-container {
```

Comment:
Please fix formatting. The file is probably using 4 spaces.

Reply:
formating fixed
![image](https://user-images.githubusercontent.com/46903421/177152554-b9d1bc8d-c57b-41a0-80ad-2465d6c7d905.png)